gtk/Application.cc

```diff
@@ -697,20 +697,10 @@ void Application::on_open(std::vector<Glib::RefPtr<Gio::File>> const& files, Gli
     impl_->open_files(files);
 }
 
-namespace
-{
-
-std::string get_application_id(std::string const& config_dir)
-{
-    struct stat sb = {};
-    (void)::stat(config_dir.c_str(), &sb);
-    return fmt::format("com.transmissionbt.transmission_{}_{}", sb.st_dev, sb.st_ino);
-}
-
-} // namespace
-
 Application::Application(std::string const& config_dir, bool start_paused, bool start_iconified)
-    : Gtk::Application(get_application_id(config_dir), TR_GIO_APPLICATION_FLAGS(HANDLES_OPEN))
+    : Gtk::Application(
+          "com.transmissionbt.transmission.gtk",
```

Comment:
I'd be interested in testing this more (but haven't had a chance yet). E.g. does it prevent 2 client instances using 2 different configuration directories from running simultaneously on the same machine under the same user? Under different users?